src/Service.Host/client/src/hooks/usePost.js

```diff
@@ -37,7 +37,12 @@ function usePost(url, requiresAuth = false, redirectOnSuccess = false) {
         const response = await fetch(id ? `${url}/${id}` : url, requestParameters);
 
         if (response.ok) {
-            const result = await response.json();
+            let result;
+            if (response.bodyUsed) {
+                result = await response.json();
+            } else {
+                result = 'Success';
+            }
```

Comment:
interesting - does the response not have a body in your case?

Reply:
no it's a rubbish end point in old stores (at the moment)